conf.yaml

```diff
@@ -2463,6 +2463,20 @@ contents:
               -
                 repo:   docs
                 path:   shared/attributes.asciidoc
+          - title:      Logstash and Kubernetes
+            prefix:     en/logstash-kubernetes
+            current:    main
+            branches:   [ {main: master} ]
+            index:      docsk8s/index.asciidoc
+            tags:       Logstash/Kubernetes
+            subject:    Logstash
+            sources:
+              -
+                repo:   logstash
+                path:   docsk8s
+              -
+                repo:   docs
+                path:   shared/attributes.asciidoc
```

Comment:
@kilfoyle, you are amazing!  I was just explaining to dev what has to happen to get this content live. I said that we need to relocate the book to "Docs in Development" but I see that you have already done that. Thank you. <3